src/Microsoft.AspNetCore.Sockets.Http/Transports/WebSocketsTransport.cs

```diff
@@ -88,23 +79,23 @@ public WebSocketsTransport(WebSocketOptions options, IChannelConnection<Message>
 
                 // Shutting down because we received a close frame from the client.
                 // Complete the input writer so that the application knows there won't be any more input.
-                _logger.LogDebug("Client closed connection with status code '{0}' ({1}). Signaling end-of-input to application", receiving.Result.Status, receiving.Result.Description);
+                _logger.LogDebug("Client closed connection with status code '{0}' ({1}). Signaling end-of-input to application", receiving.Result.CloseStatus, receiving.Result.CloseStatusDescription);
```

Comment:
I know this was already wrong when you started, but update this one to use named format specifiers too.